prime-router/docs/standard-operating-procedures/load-testing/README.md

```diff
@@ -0,0 +1,18 @@
+# Load Testing
+
+## Load Testing scripts
+
+All the load testing scripts need to have `{REPORTS_FUNCTION_KEY}` replaced with a function key for the reports 
+endpoint.
+
+- `simple-report-config.jmx`: this script sends reports with 1, 50 and 200 FHIR items; they are paced approximately so
+that there are 4x 1 item reports as 50 and 4x 50 item reports as 200
```

Comment:
I'm not sure I understand what "4x 1 item reports as 50 and 4x 50 item reports as 200" means.... I would have interpreted "4x 50 item reports as 200" as it's sending 4 reports with 50 items each to make 200, but 4 reports with 1 item each doesn't add up to 50...

Reply:
I can adjust it to be clearer, but the idea is to send 4 times as many 1 item reports as 50 item reports and 4x as many 50 item reports as 200 item reports

Reply:
Ohhh okay, that makes sense, yeah, I think if you add "as many" in there, that makes it more clear